src/V3Randomize.cpp

```diff
@@ -751,10 +754,10 @@ class ConstraintExprVisitor final : public VNVisitor {
             } else if (actual_width <= 64) {
                 fmt = "#x%16x";
             } else {
-                nodep->v3warn(CONSTRAINTIGN,
-                              "Unsupported: Associative array index "
-                              "widths of more than 64 bits during constraint randomization.");
-                return;
+                fmt = "#x%"
+                      + std::to_string((actual_width % 32 == 0) ? (actual_width / 4)
+                                                                : 8 * (int(actual_width / 32) + 1))
```

Comment:
Perhaps VL_WORDS_I?  Anyhow should be using one of the macros like that, not math here.  Similar comment above.